EventsExpress/Controllers/AuthenticationController.cs

```diff
@@ -181,23 +186,20 @@ public async Task<IActionResult> Register(LoginDto authRequest)
         public async Task<IActionResult> PasswordRecovery(string email)
         {
             if (string.IsNullOrEmpty(email))
-            {
+                {
```

Comment:
Do not format this code in such way. Block statements are always on indentation of the conditional/loop statements. Statements inside blocks have indentation = block indentation + 1.

Reply:
refactored